src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+body {
+  margin-top: 20px;
+}
+
+::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.searchbar {
```

Comment:
if you pay closer attention to the text color you'll find it's not a default one.
Please take a closer look.